acceptance/acceptance_test.go

```diff
@@ -0,0 +1,87 @@
+// +build acceptance
+
+package acceptance
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	h "github.com/buildpack/lifecycle/testhelpers"
+)
+
+func TestAcceptance(t *testing.T) {
+	spec.Run(t, "acceptance", testAcceptance, spec.Sequential(), spec.Report(report.Terminal{}))
+}
+
+func testAcceptance(t *testing.T, when spec.G, it spec.S) {
+
+	when("All", func() {
```

Comment:
Why use two levels of nesting here?

Reply:
There will be later sections specific to each binary, but since this was common to all, we chose to group them here.